src/app/[locale]/generate/hooks/useContentGeneration.ts

```
@@ -222,12 +209,19 @@ export const useContentGeneration = ({
 
             switch (contentTypeResource) {
                 case RESOURCE_CONTENT_TYPE.FILE:
-                    //TODO: Move upload file here
-                    // await ContentCreationService.insertContentTopic({
-                    //     topicId: topicId!,
-                    //     contentType: RESOURCE_CONTENT_TYPE.FILE,
-                    //     payload: {},
-                    // });
+                    const file = filesImport[0];
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)